code/modules/mob/living/carbon/human/species/plasmaman.dm

```diff
@@ -171,3 +171,23 @@
 		return FALSE //Handling reagent removal on our own. Prevents plasma from dealing toxin damage to Plasmamen.
 
 	return ..()
+
+/datum/species/plasmaman/after_equip_job(datum/job/J, mob/living/carbon/human/H)
+	if(!H.mind || !H.mind.assigned_role || H.mind.assigned_role != "Clown" && H.mind.assigned_role != "Mime")
+		H.unEquip(H.wear_mask)
+
+	H.equip_or_collect(new /obj/item/clothing/mask/breath(H), slot_wear_mask)
+	var/tank_pref = H.client && H.client.prefs ? H.client.prefs.active_character.speciesprefs : null
+	var/obj/item/tank/internal_tank
+	if(tank_pref)//Diseasel, here you go
```

Comment:
```suggestion
	if(tank_pref) //Diseasel, here you go
```
wdym by diseasel here anyway?

Reply:
Possibly copypasted from the vox one

Reply:
Used some of the Vox Code as they are the only other species that use large tank variants in pref code. I must have glared entirely over that part so that is my bad as it is Old Code. Didn't want to have two different ways of getting the same outcome, especially for a small tweak like this. 

Possibly may redefine how character prefs work in the future but that may be a while away.